ports/stm32/boards/NUCLEO_H7A3ZI/mpconfigboard.h

```diff
@@ -0,0 +1,135 @@
+#define MICROPY_HW_BOARD_NAME       "NUCLEO-H7A3ZI-Q"
+#define MICROPY_HW_MCU_NAME         "STM32H7A3ZI"
+
+#define MICROPY_HW_ENABLE_RTC       (1)
+#define MICROPY_HW_ENABLE_RNG       (1)
+#define MICROPY_HW_ENABLE_ADC       (1)
+#define MICROPY_HW_ENABLE_DAC       (1)
+#define MICROPY_HW_ENABLE_USB       (1)
+#define MICROPY_HW_ENABLE_SDCARD    (0)
+#define MICROPY_HW_HAS_SWITCH       (1)
+#define MICROPY_HW_HAS_FLASH        (1)
+
+#define MICROPY_HW_ENABLE_INTERNAL_FLASH_STORAGE (1)
+
+// The board has a 24MHz HSE, the following gives 280MHz CPU speed
+#define MICROPY_HW_CLK_PLLM (1)
+#define MICROPY_HW_CLK_PLLN (70)
+#define MICROPY_HW_CLK_PLLP (2)
+#define MICROPY_HW_CLK_PLLQ (14)
+#define MICROPY_HW_CLK_PLLR (14)
+#define MICROPY_HW_CLK_PLLVCI       (RCC_PLL1VCIRANGE_1)
+#define MICROPY_HW_CLK_PLLVCO       (RCC_PLL1VCOWIDE)
+#define MICROPY_HW_CLK_PLLFRAC      (0)
+
+//#define MICROPY_HW_CLK_PLLM         (12)
+//#define MICROPY_HW_CLK_PLLN         (280)
+//#define MICROPY_HW_CLK_PLLP         (2)
+//#define MICROPY_HW_CLK_PLLQ         (2)
+//#define MICROPY_HW_CLK_PLLR         (2)
+//#define MICROPY_HW_CLK_PLLVCI       (RCC_PLL1VCIRANGE_1)
+//#define MICROPY_HW_CLK_PLLVCO       (RCC_PLL1VCOWIDE)
+//#define MICROPY_HW_CLK_PLLFRAC      (0)
```

Comment:
Please remove commented-out code (unless it's for some reason useful, then wrap it in `#if 0`/`#endif` and add a comment about it.

Reply:
removed commented code